frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -158,66 +158,66 @@ const en: Translations = {
           modified: 'Last modified'
         },
         attachmentBox: {
-          nb: 'Huom!',
+          nb: 'Please note!',
           headline:
-            'Jos lisäät liitteet seuraaviin kohtiin sähköisesti, hakemuksesi käsitellään nopeammin, sillä käsittelyaika alkaa liitteiden saapumisesta.',
-          urgency: 'Hakemus on kiireellinen',
-          shiftCare: 'Ilta- ja vuorohoito',
-          goBackLinkText: 'Palaa takaisin hakemusnäkymään',
-          goBackRestText: 'lisätäksesi liitteet hakemukseen.'
+            ' If you add attachments to the following sections online, your application will be processed more quickly as the processing period will begin once we have received the required attachments.',
```

Comment:
Extra whitespace in the beginning of the string, intentional?
EDIT: same for the rest of these with extra whitespace; would it not be better handled in layout instead of the translations?

Reply:
Sure it would! I'd go for `&nbsp;` but do you see a better way?

Reply:
If it doesn't explicitly have to be non-breaking, there's always `{' '}`

Reply:
OK perhaps we would be better off with `{' '}`